modular_skyrat/modules/primitive_cooking_additions/code/stone_oven.dm

```diff
@@ -6,6 +6,7 @@
 	icon = 'modular_skyrat/modules/primitive_cooking_additions/icons/stone_kitchen_machines.dmi'
 	circuit = null
 	use_power = FALSE
+	flags_1 = NODECONSTRUCT_1
```

Comment:
This shouldn't have that flag, it can be constructed in-game, adding a machine that can't be deconstructed after it was built sounds like a bad idea.

Reply:
I forgot to add the part that was supposed to go with this that makes them be taken apart with a crowbar, the thing is that this isn't a machine, its an oven made of stone, it'd be REALLY fucking weird if you could screwdriver and crowbar it into a machine frame like anything else.

Reply:
It can be taken apart now, but nodeconstruct stays because it should not be spawning machine frames and such when taken apart or destroyed